doc/source/conf.py

```diff
@@ -98,6 +98,16 @@
 # copybutton config: strip console characters
 copybutton_prompt_text = r">>> |\.\.\. |\$ |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
 copybutton_prompt_is_regexp = True
+# numpydoc configuration
+numpydoc_xref_param_type = True
+numpydoc_xref_ignore = {
+    "of", "or", "optional", "default", "1D", "2D", "3D", "n-dimensional", "M", "N", "K",
+}
+numpydoc_xref_aliases = {
```

Comment:
Add two more aliases here inside `numpydoc_xref_aliases`

```
"matplotlib_axes": ":class:`matplotlib Axes <matplotlib.axes.Axes>`",
"bokeh_figure": ":bokeh:`Bokeh Figure <bokeh.plotting.Figure>`",
```

they are the ones that the other comment uses in the `ax` parameter